pennylane/spin/lattice.py

```diff
@@ -33,12 +33,13 @@ class Lattice:
     Args:
        n_cells (list[int]): Number of cells in each direction of the grid.
        vectors (list[list[float]]): Primitive vectors for the lattice.
-       positions (list[list[float]]): Initial positions of spin sites. Default value is
+       positions (list[list[float]]): Initial positions of the lattice nodes. Default value is
```

Comment:
The new version is fine, just curious why this is changed?

Reply:
Lattice nodes are not necessary the spin sites. In many models, spins live on the middle of edge.

Reply:
There was a sub-thread in Slack thread where @ddhawan11 and I discussed - its because both "spin sites" and "lattice nodes" were used in different arguments in the docstring to mean the same thing, so for clarity we are going for a single term for a single concept.

Reply:
Thanks, all good.

Reply:
Lattice nodes are definitely better choice since conceptually lattice is independent of spin and here we're defining lattice.